.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.7
```

Comment:
Why are we specifying this version in particular?

Reply:
Doesn't particularly matter, since Ruby isn't used too much in this project. I'll raise it to a higher and not deprecated version.